<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeComposition</class>
 <widget class="QMainWindow" name="MakeComposition">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>758</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Lucida Sans</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Оформление поста</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
font: 10pt &quot;Lucida Sans&quot;;</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	align-text: center;
	
	background-color: #F8E6E0;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="main_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget {
	background-color: #F8E6E0;
	border-radius: 10px;
}</string>
      </property>
      <layout class="QGridLayout" name="main">
       <item row="0" column="0">
        <widget class="QFrame" name="input_widget">
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: #ECCEF5;
	padding: 10px;
	height: 100%;
}</string>
         </property>
         <layout class="QGridLayout" name="form_input">
          <item row="3" column="1">
           <widget class="QLineEdit" name="main_genre_value">
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	font: 75 10pt Lucida Sans;
	color: #2121F2;
	border: 1px solid gray;
	border-radius: 5px;
	background-color: rgb(255, 255, 255);
	padding: 4px;
}
</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="text_composition_label">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font: 75 10pt Lucida Sans;
	color: #2121F2;
	border: 1px solid gray;
	border-radius: 5px;
	background-color: #F6D8CE;
}</string>
            </property>
            <property name="text">
             <string>Текст произведения:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="name_label">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font: 75 10pt Lucida Sans;
	color: #2121F2;
	border: 1px solid gray;
	border-radius: 5px;
	background-color: #F6D8CE;
}</string>
            </property>
            <property name="text">
             <string>Название произведения</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="age_limit_label">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font: 75 10pt Lucida Sans;
	color: #2121F2;
	border: 1px solid gray;
	border-radius: 5px;
	background-color: #F6D8CE;
}</string>
            </property>
            <property name="text">
             <string>Ограничение возраста:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QComboBox" name="age_limit_box">
            <property name="styleSheet">
             <string notr="true">QComboBox {
	font: 75 10pt Lucida Sans;
	color: #2121F2;
	border: 1px solid gray;
	border-radius: 5px;
	background-color: rgb(255, 255, 255);
	padding: 4px;
}

</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QTextEdit" name="text_composition_value">
            <property name="styleSheet">
             <string notr="true">QTextEdit {
	font: 75 10pt Lucida Sans;
	color: #2121F2;
	border: 1px solid gray;
	border-radius: 5px;
	background-color: rgb(255, 255, 255);
}</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="genre_label">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font: 75 10pt Lucida Sans;
	color: #2121F2;
	border: 1px solid gray;
	border-radius: 5px;
	background-color: #F6D8CE;
}</string>
            </property>
            <property name="text">
             <string>Жанр:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="name_value">
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	font: 75 10pt Lucida Sans;
	color: #2121F2;
	border: 1px solid gray;
	border-radius: 8px;
	background-color: rgb(255, 255, 255);
	padding: 8px;
}
</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="button_form">
         <property name="styleSheet">
          <string notr="true">QFrame{
	background-color: #ECCEF5;
	padding: 5px;
}</string>
         </property>
         <layout class="QGridLayout" name="form_button">
          <item row="1" column="0">
           <widget class="QPushButton" name="back_btn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #F6D8CE;
	font: 75 10pt &quot;Lucida Sans&quot;;
	color: #2121F2;
	border: 1px solid gray;
	border-radius: 4px;
}

QPushButton:hover {
	border: 2px solid gray;
}
</string>
            </property>
            <property name="text">
             <string>Вернуться</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="send_btn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #F6D8CE;
	font: 75 10pt &quot;Lucida Sans&quot;;
	color: #2121F2;
	border: 1px solid gray;
	border-radius: 4px;
}

QPushButton:hover {
	border: 2px solid gray;
}</string>
            </property>
            <property name="text">
             <string>Отправить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="information_label">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font: 75 10pt Lucida Sans;
	color: #2121F2;
}</string>
            </property>
            <property name="text">
             <string>Автосохранение работает.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
